mlflow/R/mlflow/tests/testthat/test-model-registry.R

```diff
@@ -75,6 +75,68 @@ test_that("mlflow can register a model with tags and description", {
   )
 })
 
+test_that("mlflow can get a registered model", {
+  with_mock(.env = "mlflow",
+    mlflow_rest = function(...) {
+      args <- list(...)
+      expect_true(paste(args[1:2], collapse = "/") == "registered-models/get")
+      expect_equal(args$verb, "GET")
+      expect_equal(args$query$name, "test_model")
+      return(list(
+        registered_model = list(
+          name = "test_model"
+        )
+      ))
+    }, {
+      mock_client <- get_mock_client()
+
+      mlflow_get_registered_model("test_model", client = mock_client)
+  })
+})
+
+test_that("mlflow can rename a registered model", {
+  with_mock(.env = "mlflow",
+    mlflow_rest = function(...) {
+      args <- list(...)
+      expect_true(paste(args[1:2], collapse = "/")
+                  == "registered-models/rename")
+      expect_equal(args$verb, "POST")
+      expect_equal(args$data$name, "old_model_name")
+      expect_equal(args$data$new_name, "new_model_name")
+
+      return(list(
+        registered_model = list(
+          name = "new_model_name"
+        )
+      ))
+    }, {
+      mock_client <- get_mock_client()
+      mlflow_rename_registered_model("old_model_name", "new_model_name",
+                                     client = mock_client)
+  })
+})
+
+test_that("mlflow can update a model", {
+  with_mock(.env = "mlflow",
+    mlflow_rest = function(...) {
+      args <- list(...)
+      expect_true(paste(args[1:2], collapse = "/")
```

Comment:
You could use `expect_equivalent()` to avoid the line break around `==`.

Reply:
Done. Used `expect_equal` instead